packages/paima-sdk/paima-utils/src/config.ts

```diff
@@ -157,6 +157,10 @@ export class ENV {
     return process.env.MQTT_BATCHER_BROKER_URL || 'ws://127.0.0.1:' + ENV.MQTT_BATCHER_BROKER_PORT;
   }
 
+  static get GET_LOGS_MAX_BLOCK_RANGE(): number {
+    return parseFloat(process.env.GET_LOGS_MAX_BLOCK_RANGE || '5000');
```

Comment:
Hmm, do we need this? Seems a little arbitrary. If there's a rationale for this, we should definitely document it

Reply:
the number is rather arbitrary, that said, I don't think it's generally a good idea to have unbounded queries? eth based nodes usually have a max range too. My concern is mostly DoS though. If this is covered by some other factor (a proxy?) I'll remove this. But isn't the engine's server usually queried by the clients directly?

Reply:
By default nodes are bounded, but archive nodes aren't bounded and those are usually what you get when you pay for an RPC provider. 

That is to say, I don't necessarily think have a default limit is bad (and you can have nodes that people pay to access if they want more historical data), but the reason Ethereum does it this was is because light nodes don't store this data at all (so you can't even get this data if you want to), which is different from our case where the node has all the logs anyways and is just refusing to serve them to you

i.e. I think our Ethereum equivalent would be to do something like say that if `STORE_HISTORICAL_GAME_INPUTS` is false, we only keep 5000 logs in the database or something like that

Reply:
I see. I'm not sure why, but I was sure I had issues in the past with requesting logs from too many blocks in a single request (like for example, when setting the `funnelBlockGroupSize` too large in the evm funnel settings). And I also think I had problems with Besu at some point where I had to shrink the block size or it just didn't return anything, but maybe it was something else.

Anyway, maybe this is the wrong approach and instead we may want to paginate this endpoint (with a default max page size)? Although not sure who that would be adapted in the eth rpc wrapper then.